tools/setup/emoji/emoji_names.py

```diff
@@ -1577,4 +1577,1910 @@
+        "aliases": ["baby", "breast", "breast-feeding", "nursing"],
+    },
+    "1f932": {"canonical_name": "palms up together ", "aliases": ["palms up together", "prayer"]},
+    "1f935-200d-2640": {
+        "canonical_name": "woman in tuxedo ",
+        "aliases": ["tuxedo", "woman", "woman in tuxedo"],
+    },
+    "1f935-200d-2642": {
+        "canonical_name": "man in tuxedo ",
+        "aliases": ["man", "man in tuxedo", "tuxedo"],
+    },
+    "1f937-200d-2640": {
+        "canonical_name": "woman shrugging ",
+        "aliases": ["doubt", "ignorance", "indifference", "shrug", "woman", "woman shrugging"],
+    },
+    "1f937-200d-2642": {
+        "canonical_name": "man shrugging ",
+        "aliases": ["doubt", "ignorance", "indifference", "man", "man shrugging", "shrug"],
+    },
+    "1f938-200d-2640": {
+        "canonical_name": "woman cartwheeling ",
+        "aliases": ["cartwheel", "gymnastics", "woman", "woman cartwheeling"],
+    },
+    "1f938-200d-2642": {
+        "canonical_name": "man cartwheeling ",
+        "aliases": ["cartwheel", "gymnastics", "man", "man cartwheeling"],
+    },
+    "1f939-200d-2640": {
+        "canonical_name": "woman juggling ",
+        "aliases": ["juggling", "multitask", "woman"],
+    },
+    "1f939-200d-2642": {
+        "canonical_name": "man juggling ",
+        "aliases": ["juggling", "man", "multitask"],
+    },
+    "1f93c-200d-2640": {
+        "canonical_name": "women wrestling ",
+        "aliases": ["women", "women wrestling", "wrestle"],
+    },
+    "1f93c-200d-2642": {
+        "canonical_name": "men wrestling ",
+        "aliases": ["men", "men wrestling", "wrestle"],
+    },
+    "1f93d-200d-2640": {
+        "canonical_name": "woman playing water polo ",
+        "aliases": ["water polo", "woman", "woman playing water polo"],
+    },
+    "1f93d-200d-2642": {
+        "canonical_name": "man playing water polo ",
+        "aliases": ["man", "man playing water polo", "water polo"],
+    },
+    "1f93e-200d-2640": {
+        "canonical_name": "woman playing handball ",
+        "aliases": ["handball", "woman", "woman playing handball"],
+    },
+    "1f93e-200d-2642": {
+        "canonical_name": "man playing handball ",
+        "aliases": ["handball", "man", "man playing handball"],
+    },
+    "1f93f": {
+        "canonical_name": "diving mask ",
+        "aliases": ["diving", "diving mask", "scuba", "snorkeling"],
+    },
+    "1f94c": {"canonical_name": "curling stone ", "aliases": ["curling stone", "game", "rock"]},
+    "1f94d": {"canonical_name": "lacrosse ", "aliases": ["ball", "goal", "lacrosse", "stick"]},
+    "1f94e": {"canonical_name": "softball ", "aliases": ["ball", "glove", "softball", "underarm"]},
+    "1f94f": {"canonical_name": "flying disc ", "aliases": ["flying disc", "ultimate"]},
+    "1f95f": {
+        "canonical_name": "dumpling ",
+        "aliases": ["dumpling", "empanada", "gyōza", "jiaozi", "pierogi", "potsticker"],
```

Comment:
Hmm, I wonder if our emoji search tries a diacritics-removed match, so that "gyoza" would match this. I guess it should!

Similarly "piña colada" below, and a few of the territory names like "Côte d’Ivoire".

Reply:
Seems like it didn't! I manually added some aliases, but that's not the most ideal approach.

Reply:
Discussion here: https://chat.zulip.org/#narrow/stream/6-frontend/topic/update.20emoji.20.2319371/near/1392206